charts/kommander-operator/templates/rback_v1_clusterrole_operator_kommander-operator.yaml

```diff
@@ -58,6 +59,7 @@ rules:
       - kustomize.toolkit.fluxcd.io
       - types.kubefed.io
       - kommander.mesosphere.io
+      - ""
```

Comment:
what's the effect of this empty string apiGroup?

Reply:
`""` --> core api group
https://kubernetes.io/docs/reference/using-api/#api-groups
> The core (also called legacy) group is found at REST path /api/v1. The core group is not specified as part of the apiVersion field, for example, apiVersion: v1.